### ISSUE TITLE
minikube add-on mgr can't be retrieved

### ISSUE THREAD

**Issue by orefalo** [superbrothers/minikube-ingress-dns#5]
weird issue, when I use this great extension. I get those...

```
NAME                               READY   STATUS         RESTARTS   AGE
coredns-5644d7b6d9-92jwf           1/1     Running        0          104s
coredns-5644d7b6d9-zdwtl           1/1     Running        0          104s
etcd-minikube                      1/1     Running        0          50s
kube-addon-manager-minikube        0/1     ErrImagePull   0          104s
kube-apiserver-minikube            1/1     Running        0          43s
kube-controller-manager-minikube   1/1     Running        0          40s
kube-proxy-fkcd8                   1/1     Running        0          103s
kube-scheduler-minikube            1/1     Running        0          31s
```

```
minikube start
😄  minikube v1.5.2 on Darwin 10.15.1
✨  Automatically selected the 'hyperkit' driver (alternates: [vmwarefusion])
🔥  Creating hyperkit VM (CPUs=2, Memory=2000MB, Disk=20000MB) ...
⚠️  VM may be unable to resolve external DNS records
⚠️  VM is unable to access k8s.gcr.io, you may need to configure a proxy or set --image-repository
🐳  Preparing Kubernetes v1.16.2 on Docker '18.09.9' ...
🚜  Pulling images ...
❌  Unable to pull images, which may be OK: running cmd: "/bin/bash -c \"sudo env PATH=/var/lib/minikube/binaries/v1.16.2:$PATH kubeadm config images pull --config /var/tmp/minikube/kubeadm.yaml\"": Process exited with status 1
🚀  Launching Kubernetes ...
⌛  Waiting for: apiserver
🏄  Done! kubectl is now configured to use "minikube"
```

Not how one of the images can't be pulled

Issue also explored here.. https://github.com/kubernetes/minikube/issues/5924

**Comment by orefalo** [comment]
Looks like this is the issue

https://github.com/kubernetes/minikube/issues/3104

**Comment by orefalo** [comment]
found this issue, dnsmasq can't be running on port 53, will submit pullreq

**Comment by superbrothers** [comment]
Closed by #12
